docs/learn/bitesize/bitesize-socialhub.md

```diff
@@ -1,30 +1,32 @@
 ---
 
-The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project aimed to demonstrate Distributed Authority's core features and benefits, helping users seamlessly integrate Distributed Authority into their own game projects.
-Within Social Hub, you are free to explore Distributed Authority's key features & functionalities in a sandbox-like environment. Such features include:
+The [Distributed Authority Social Hub Sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/DistributedAuthoritySocialHub) is a project that demonstrates Distributed Authority's features and helps you integrate Distributed Authority into your own game projects.
 
-* Automatic redistribution of ownership of NetworkObjects across connected clients
-* Client-side NetworkObject spawning and deferred NetworkObject despawning
-* Ownership transfer of NetworkObjects with variable ownership flags
-* Host migration of a networked game session
+Within Social Hub, you can explore Distributed Authority in a sandbox-like environment. You can:
+
+- Automatically redistribute ownership of NetworkObjects across connected clients
+- Client-side NetworkObject spawning and deferred NetworkObject despawning
+- Transfer ownership of NetworkObjects with variable ownership flags
+- Host migration of a networked game session
 
 ## The Bootstrap scene
 
-This is the entry point to the sample. Once loaded, you will be transitioned to the MainMenu scene.
+This is the entry point to the sample. After it loads, the MainMenu scene appears.
 
 ## The MainMenu scene
 
-Here, you're able to enter a player name, as well as a session name. When these fields are populated, you may then create or join an existing game session by pressing the *Start* button.
+1. Enter a player name and a session name.
+2. Select **Start** to create or join an existing game.
 
-## The HubScene_TownMarket Scene
+## The HubScene_TownMarket scene
 
-You'll be placed in a sandbox-like environment that supports 64-players. The control model is WASD controls, with the *E* button mapped as the key to interact with crates and pots. These NetworkObjects can be picked up and subsequently dropped or thrown, showcasing how to attach one NetworkObject to another with an animated rig through the use of FixedJoints. As a NetworkObject is picked up by a client, authority of that NetworkObject is transferred to that client.
+This a sandbox-like environment that supports 64 players. The control model is WASD controls, with the *E* button mapped as the key to interact with crates and pots. These are NetworkObjects that you can pick up, drop, and throw. Attach one NetworkObject to another with an animated rig through the use of FixedJoints. As a client picks up a NetworkObject, authority of that NetworkObject transfers to that client.
```

Comment:
What is/are the main task(s) a user will do with this scene?
Side note: I could not get this to load nicely, either.

Reply:
Oh, even when adding a cloud project id, and starting the flow from the Bootstrap scene?

As for what you do when properly loaded, it's a sandbox, so there's really no goal or end condition to the sample. Players are basically able to move around the level with client-authority, and interact with these objects that implement the basic Distributed Authority features listed above.

We could also highlight how this game inherently supports host migration out of the box. One can load multiple clients, and witness ownership of NetworkObjects be automatically distributed when the original session owner leaves the session.